### PR TITLE
Update README for AlmaLinux

### PR DESCRIPTION
**Pull request by prilr**

### CODE REVIEW

**Comment by shakaran** [comment]
@prilr could you rebase again the conflicts? Thanks

**Comment by prilr** [comment]
@shakaran Done.

**Comment by shakaran** [comment]
@pirat89 @codyro @andrewlukoshko @bennyvasquez could you merge this? Thanks

File: README.md [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
+## Running
+Make sure your system is fully updated before starting the upgrade process.
+
+```bash
+sudo yum update -y
+```
+
+Install `elevate-release` package with the project repo and GPG key.
+
+`sudo yum install -y http://repo.almalinux.org/elevate/elevate-release-latest-el7.noarch.rpm`
```
Comment (andrewlukoshko) [inline README.md:14]: Replace with:
```
sudo yum install -y http://repo.almalinux.org/elevate/elevate-release-latest-el$(rpm --eval %rhel).noarch.rpm
```

**Review by andrewlukoshko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (prilr) [reply README.md]: Done.

**Pull request merged**